packages/lib-classifier/package.json

```diff
@@ -45,7 +45,7 @@
   },
   "peerDependencies": {
     "@zooniverse/grommet-theme": "3.x.x",
-    "@zooniverse/panoptes-js": "~0.1.0",
+    "@zooniverse/panoptes-js": "~0.1.2",
```

Comment:
[minor query] Why are some of these dependencies using `~0.1.2` while others use `0.1.2` ?

Reply:
Dangit, my review forgot to also ask why `"url-parse": "1.5.10"` lost the `~` in the version specification.

Reply:
`~0.1.0` includes `0.1.2`. 🤔